tests/Feature/SettingsTest.php

```diff
@@ -0,0 +1,74 @@
+<?php
+
+/**
+ * We don't care for unhandled exceptions in tests.
+ * It is the nature of a test to throw an exception.
+ * Without this suppression we had 100+ Linter warning in this file which
+ * don't help anything.
+ *
+ * @noinspection PhpDocMissingThrowsInspection
+ * @noinspection PhpUnhandledExceptionInspection
+ */
+
+namespace Tests\Feature;
+
+use App\DTO\SortingCriterion;
+use App\Facades\AccessControl;
+use App\Http\Requests\Settings\SetSortingRequest;
+use Tests\TestCase;
+
+class SettingsTest extends TestCase
+{
+	public function testSetSorting(): void
```

Comment:
Again, please split this method. I see at least

- `testSetSorting` (the postiive case)
- `testSetSortingWithIllegalAlbumAttribute`
- `testSetSortingWithIllegalPhotoAttribute`
- `testSetSortingWithUnknownOrder`